backend/src/Edna/Orphans.hs

```diff
@@ -15,6 +15,12 @@ import Servant.Util.Common.Common (ApiHasArgClass(..))
 -- so it's simpler to just have one line for now.
 deriving newtype instance MonadCatch (RIO env)
 
+-- It's better to use @throw@ family of functions instead of @fail@, but
+-- in tests we want to do incomplete pattern-matching sometimes and @MonadFail@
+-- is needed for that.
+instance MonadFail (RIO env) where
```

Comment:
Looks like, if you use `readCreateProcessWithExitCode`, this instance won't be needed outside of tests.